src/jsonminifier.cpp

```diff
@@ -38,6 +36,7 @@ static uint8_t jump_table[256 * 3] = {
     0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1, 0, 1, 1,
 };
 
+// This is a (slow) portable minifier.
```

Comment:
Assuming we're satisfied with the implementation (I think we are), I'd actually move this into `fallback->minify()` and make the core `json_minify()` method call `active_implementation->minify()`.

Reply:
Ok. I will do that.

Reply:
I just removed json_minify entirely. Well, we still have the implementation, but it is now entirely in fallback, where it belongs.